hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/HFile.java

```diff
@@ -316,7 +316,6 @@ public Writer create() throws IOException {
           ostream.setDropBehind(shouldDropBehind && cacheConf.shouldDropBehindCompaction());
         } catch (UnsupportedOperationException uoe) {
           LOG.trace("Unable to set drop behind on {}", path, uoe);
```

Comment:
this is an idiom in the code base where we want something logged at one level but whoever figured we don't usually need the stacktrace with it, so the same message gets logged at a lower level with it.

in this case you should keep it at DEBUG and add the stacktrace if desired.